portal/templates/contract/contract-view.njk

```diff
@@ -68,118 +68,110 @@
-                  Bond transactions
-                </span>
-            <span class="govuk-!-margin-left-6">
-              {{ statusTag.render(deal.bondTransactions.status) }}
-            </span>
-          </h2>
-        </div>
+      {% if deal.details.manualInclusionNoticeSubmissionDate %}
+        <span class="govuk-heading-m govuk-grid-column-one-quarter govuk-!-font-weight-regular govuk-!-font-size-16 govuk-!-padding-right-0 govuk-!-padding-left-0 width-auto"  data-cy="manual-inclusion-notice-submission-date">
+          <span class="govuk-!-font-weight-bold">Submission date: </span>
+          <br/>
+          {{ deal.details.manualInclusionNoticeSubmissionDate | localiseTimestamp('DD/MM/YYYY', user.timezone) }}
+        </span>
+      {% endif %}
 
-        <div class="govuk-grid-column-one-quarter box__header-link">
-          {{ addABondButton.render(componentData) }}
-        </div>
+      {% if deal.details.submissionDate %}
+        {% set submissionDateClass = "govuk-grid-column-one-quarter govuk-!-padding-right-0 box__header-link width-auto" %}
 
-        {% if deal.bondTransactions.items.length %}
-          {{ bondTransactionsTable.render(componentData) }}
-        {% endif %}
+      {% else %}
+        {% set submissionDateClass = "govuk-grid-column-one-quarter box__header-link" %}
+      {% endif %}
 
+      <div class="{{ submissionDateClass }}">
+        {{ eligibilityCriteriaLink.render(componentData) }}
       </div>
     </div>
 
     <div class="govuk-grid-row govuk-!-padding-top-2 govuk-!-padding-bottom-2 box">
-      <div class="box__header">
-        <div class="govuk-grid-column-three-quarters">
-          <h2 class="govuk-heading-m govuk-!-margin-bottom-0">
-            <span class="govuk-grid-column-one-half govuk-!-padding-left-0">
-                  Loan transactions
-                </span>
-
-            <span class="govuk-!-margin-left-6">
-              {{ statusTag.render(deal.loanTransactions.status) }}
-            </span>
+      <div class="govuk-grid-column-three-quarters">
+        <h2 class="govuk-grid-row govuk-heading-m govuk-!-margin-bottom-0">
+          <span class="govuk-grid-column-one-half">
+            Bond transactions
+          </span>
+          <span class="govuk-grid-column-one-half">
+            {{ statusTag.render(deal.bondTransactions.status) }}
+          </span>
+        </h2>
+      </div>
+
+      <div class="govuk-grid-column-one-quarter box__header-link">
+        {{ addABondButton.render(componentData) }}
+      </div>
+
+      {% if deal.bondTransactions.items.length %}
+        {{ bondTransactionsTable.render(componentData) }}
+      {% endif %}
+
+    </div>
 
-          </h2>
-        </div>
+    <div class="govuk-grid-row govuk-!-padding-top-2 govuk-!-padding-bottom-2 box">
+      <div class="govuk-grid-column-three-quarters">
+        <h2 class="govuk-grid-row govuk-heading-m govuk-!-margin-bottom-0">
```

Comment:
Added class `govuk-grid-row` so that the two spans are a subgrid inside this header.